docs/FeatureManagement/Clients/SpringCloud/Spring.FeatureFlag.v1.0.0.schema.json

```diff
@@ -0,0 +1,89 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://azconfig.io/schemas/FeatureManagement/v1.0.0/SpringFeatureFlag.json",
```

Comment:
This looks like it would be better as `http://azconfig.io/schemas/FeatureManagement/Spring/v1.0.0/FeatureManagement.json`

Reply:
@mrm9084, the ending "SpringFeatureFlag.json" doesn't seem right. This is not the schema for a feature flag. It is the schema for the feature management section in a Spring Cloud application's configuration. That's why I suggested "http://azconfig.io/schemas/FeatureManagement/Spring/v1.0.0/FeatureManagement.json"

Reply:
I see that it was halfway updated, it looks like just the last part was missed.